### DISCUSSION TITLE
Will Dawn soon allow media grouping per variant?

### DISCUSSION THREAD

**Discussion by teezecrost1179** [Shopify/dawn#3563] [Q&A]
There are a number of tutorials like this one (https://www.youtube.com/watch?v=FHzBvy85ais) that allow us to assign groups of media to each variant. Will this ability be native to the theme soon? Or am I missing something and this is already possible?

**Comment by joshistoast** [comment]
The combined listings may be a better approach to this

**Comment by bakura10** [comment]
We have been doing this for a long time in our themes due to merchant request, but I think Dawn should not do it. All existing approaches are hacky, time consuming for merchants, highly inefficient, and can cause accessibility issues. This feature alone causes us a lot of support due to all the manual steps needed, so I think that at Dawn scale this would quickly make the Shopify support collapse.

the combined listing approach is the way to go, but being a Shopify Plus only app this is limited.

Shopify should really have a native solution around this issue.
